control-plane/api-gateway/common/translation_test.go

```diff
@@ -354,9 +355,10 @@ func TestTranslator_ToAPIGateway(t *testing.T) {
 func TestTranslator_ToHTTPRoute(t *testing.T) {
 	t.Parallel()
 	type args struct {
-		k8sHTTPRoute gwv1beta1.HTTPRoute
-		services     []types.NamespacedName
-		meshServices []v1alpha1.MeshService
+		k8sHTTPRoute    gwv1beta1.HTTPRoute
+		services        []types.NamespacedName
+		meshServices    []v1alpha1.MeshService
+		externalFilters map[gwv1beta1.LocalObjectReference]v1alpha1.ExternalRouteFilter
```

Comment:
should this be updated to take some arbitrary `ObjectReference` filters in and out of the namespace just to test the namespace alignment checking?